Plotly.NET.sln

```diff
@@ -83,6 +83,10 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = ".ci", ".ci", "{2461AFBF-6E1
 		.circleci\config.yml = .circleci\config.yml
 	EndProjectSection
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{FDA42E0F-C03B-47C1-B24B-EAEEFE281514}"
```

Comment:
Very minor thing: You can add a project to the solution root with dotnet sln <sln file> add --in-root <proj file> instead of also adding the `src` folder